### PR TITLE
[merge] Feature/ftp node

### PR DESCRIPTION
**Pull request by giovannicuriel**
Merge of a previous PR

### CODE REVIEW

File: orchestrator/nodes/ftp/index.js [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+                response = await client.upload(stream, filename);
+                this._set(config.response, {}, response);
+            break;
+            case "GET": {
+                response = await client.download(filename);
+                this._set(config.response, {}, response);
+            }
+        }
+        client.close();
+        return [message];
```
Comment (raulnegreiros) [inline orchestrator/nodes/ftp/index.js]: Promise.resolve([message]);

File: orchestrator/nodes/ftp/index.js [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+        logger.debug(`Config is: ${util.inspect(config)}`);
+        var url = config.url;
+        var tokens = config.url.match(/(ftp|ftps):\/\/(.*)/);
+        var transport = "ftp";
+        var host;
+        var port = 21;
+        var remaining;
+        var encoding = config.fileencoding;
+        const user = config.username;
+        const password = config.password; // THIS SHOULD NOT BE LIKE THIS!
```
Comment (raulnegreiros) [inline orchestrator/nodes/ftp/index.js]: hum? What do you mean with this comment?

Comment (giovannicuriel) [reply orchestrator/nodes/ftp/index.js]: It should be stored elsewhere in a safe way.

Comment (giovannicuriel) [reply orchestrator/nodes/ftp/index.js]: Waiting for Vault to be included.

Comment (raulnegreiros) [reply orchestrator/nodes/ftp/index.js]: agree, we must talk about it in the future.

**Pull request merged**